### PR TITLE
destination-s3-glue: Fix decimal type syntax

### PR DESCRIPTION
**Pull request by blarghmatey**
## What
The definition of the decimal type as `decimal(38,38)` was incorrect as it caused the query engine to interpret that as meaning that all 38 digits had to be to the right of the decimal place, so any whole number values would overflow. Setting it as `decimal(38)` allows for the full 38 digits, but with flexible scale.

## How
Set the decimal type creation syntax to only specify the precision, leaving the scale as a flexible value.

## Recommended reading order
1. `GlueOperations.java`

## 🚨 User Impact 🚨
Any `select *` queries using a decimal type will no longer result in `fieldValue cannot be null` errors.

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by blarghmatey** [comment]
@marcosmarxm this is hopefully the last one of these I'll need to do for now :simple_smile: Thanks again for all of your prompt assistance!

**Review by marcosmarxm — APPROVED** [review]

**Comment by blarghmatey** [comment]
@marcosmarxm is there anything I can do once a PR is approved to get the connector built and pushed?

**Comment by blarghmatey** [comment]
Would it be possible to get a build of this connector today?

**Comment by blarghmatey** [comment]
@marcosmarxm any chance of getting this built and merged before EOD? Sorry to be pushy, just trying to unblock some work for my team.

**Comment by marcosmarxm** [comment]
Published in https://github.com/airbytehq/airbyte/pull/25813

**Pull request closed without merging**
